### ISSUE TITLE
Publish to npm?

### ISSUE THREAD

**Issue by allthesignals** [jasonkriss/ember-scroll-to#40]
Is the latest version going to be published to npm soon? Thanks!

**Comment by sardyy** [comment]
@blimmer @Duder-onomy It would be really nice if you can launch a new release, I'm using it in production app :) Best regardings

**Comment by Duder-onomy** [comment]
@blimmer Getting a NPM publish will (IMO) close a bunch of issues and PRs.

**Comment by strepto42** [comment]
+1 please :)

**Comment by Duder-onomy** [comment]
@blimmer Ping^ Can we score an npm publish? 
It will close a bunch of issues and PR's

**Comment by rmachielse** [comment]
@blimmer a publish would be very much appreciated! Or @jasonkriss can you do it?

**Comment by Duder-onomy** [comment]
@allthesignals @sardyy @strepto42 @rmachielse @blimmer @jasonkriss 
Since we cannot seem to get this thing published to NPM, I have made a mirror repo and published it to NPM, it is called `ember-scroll-to-mirror`. I will keep it up to date with master over here. You can be safe using that one for now.  https://github.com/Duder-onomy/ember-scroll-to-mirror

We might want to explore a legitimate fork as we cannot get ahold of the repo owners.

**Comment by rmachielse** [comment]
@blimmer @jasonkriss another attempt, can someone please execute the simple task of running `npm publish`?

**Comment by blimmer** [comment]
I'm sorry - I'm not an owner on this so I can't publish:

```console
> npm owner ls ember-scroll-to
ragnarpeterson <ragnarpeterson1@gmail.com>
```

@ragnarpeterson is the only one.

**Comment by alanlalonde** [comment]
If this project is abandoned by its owner, someone may want to go through the npm adoption process through this link. https://www.npmjs.com/policies/disputes

**Comment by blimmer** [comment]
I sent an email today in accordance with the NPM policy disputes linked above. If the owner doesn't respond in 4 weeks, we should be able to take over ownership on the NPM side.

![screen shot 2018-11-02 at 10 49 57](https://user-images.githubusercontent.com/630449/47928923-33771f80-de8d-11e8-8e92-b30c9877d44f.png)

**Comment by allthesignals** [comment]
Thanks @blimmer

**Comment by blimmer** [comment]
NPM got back to me and I now have publish rights. I've published 0.6.5.
